tests/models/test_energy_balance.py

```diff
@@ -221,9 +221,11 @@ def bc_values_pressure(self, boundary_grid: pp.BoundaryGrid) -> np.ndarray:
 
     solid_vals = pp.solid_values.extended_granite_values_for_testing
     fluid_vals = pp.fluid_values.extended_water_values_for_testing
+    numerical_vals = {"characteristic_displacement": 0.2}
```

Comment:
@IvarStefansson What do you think how the definition of numerical constants is handled in the tests? There has been two possibilities: 1. Explicit definition in all tests (like done here). 2. A central definition as used for the material constants. To me both is fine. Yet, if I count correctly, it is 5 instances in which the characteristic displacement needs explicit setting. Thus, centralizing it would also be reasonable.

Reply:
Good question. I guess it depends on whether we expect to expand the set of numerical parameters and, if so, what they will contain. If we end up with a larger set which can readily be reused, I'd prefer the sentralised version.

Reply:
I agree. Let's go for a centralized variant already now. This will then remove the need to think about that again, when adding other values later.

Reply:
Should be fixed now.